optuna/study/study.py

```diff
@@ -322,7 +322,7 @@ def optimize(
         n_trials: Optional[int] = None,
         timeout: Optional[float] = None,
         n_jobs: int = 1,
-        catch: Tuple[Type[Exception], ...] = (),
+        catch: Union[Tuple[Type[Exception], ...], Type[Exception]] = (),
```

Comment:
Similarly, this part might be relaxed from `Tuple` to `Iterable`.
```suggestion
        catch: Union[Iterable[Type[Exception]], Type[Exception]] = (),
```